### ISSUE TITLE
Resource pages don't render alt-text for AT

### ISSUE THREAD

**Issue by tsmyre** [ManifoldScholar/manifold#2428]
It does not appear that Project Resource pages are presently configured to render accessibility metadata for assistive technology. In the example here, we have an image resource that includes alt-text. However VoiceOver is unable to focus on the resource itself nor secure the entered alt-text.

Beyond the scope of this issue alone, VoiceOver could not focus on all the elements that visually appear on the page: the title, added date, metadata in the right column, caption, full description header, and the full description itself weren't available for selection.

Resource URL: `https://manifold.umn.edu/projects/cut-copy-paste/resource/r081`
Alt-Text in Field: `Photograph of a page opening in Benlowes’s gift copy of Theophila to St. John’s College Library.`

**Comment by dananjohnson** [comment]
@tsmyre Thanks for catching this! The first issue stems from the fact that the resource image is being rendered as a `<div>` with a CSS background image. That's semantically not right—the image is a critical piece of content, not decorative—and we should fix it.

RE: the second issue, I'm not sure I follow. What do you mean when you say that these elements "weren't available for selection"? Are you trying to focus on them with the `tab` key?

**Comment by tsmyre** [comment]
@dananjohnson Yes, that's correct, I couldn't tab to those elements. As I navigated the page with `tab`, focus never landed on them.

**Comment by dananjohnson** [comment]
@tsmyre That's expected. `tab` navigates to the next focusable node, and only [interactive elements](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Interactive_content) are considered focusable.

**Comment by dananjohnson** [comment]
VoiceOver _will_ start reading the page content if you press `ctrl` + `option` + `A`.

I make regular use of this [reference guide](https://dequeuniversity.com/assets/pdf/screenreaders/voiceover-macos-guide.pdf) for using all of VoiceOver's different features.

**Comment by dananjohnson** [comment]
This issue should be solved in conjunction with #2433.

**Comment by tsmyre** [comment]
Thank you, @dananjohnson; this quick reference guide is super helpful and much appreciated!

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by Santoz2** [comment]
Accessibility Triage Notes: UW's ATX team considers this a priority 1 issue, because it appears that the major purpose of this resource page is to check that the information for a resource is correct. As it is now, it would appear to a non-visual user that the image, the resource that they are looking for, is missing. 

It is worth noting that the proper alt text is present once a user activates the "Zoom" button, but the zoom button itself does not seem to be keyboard accessible (can only seem to interact with it using a mouse).
